<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,0,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="1,0">
        <item>
         <widget class="ClickableLabel" name="label_VideoFrame">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="PB_play">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Edit Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Open Files</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QPushButton" name="PB_open_video">
                  <property name="text">
                   <string>OpenVideo</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="PB_open_json">
                  <property name="text">
                   <string>Open Json</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Replace and Delete</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="2" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>from frame:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>new_id:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>act_id:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>to frame:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="text_actId"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="text_newId"/>
                </item>
                <item row="5" column="1">
                 <widget class="QPushButton" name="PB_delete">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="text_toFrame"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="text_fromFrame"/>
                </item>
                <item row="5" column="0">
                 <widget class="QPushButton" name="PB_replace">
                  <property name="text">
                   <string>Replace</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QPushButton" name="PB_add">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Correct flickering</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_maxAge">
                  <property name="text">
                   <string>5</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_flickerin_status">
                  <property name="text">
                   <string> </string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QPushButton" name="PB_flickering">
                  <property name="text">
                   <string>Flickering</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="text_toFrameFlick"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>from frame: </string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>max_age:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="text_fromFrameFlick"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>to frame:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Face</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Ids:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_ids">
            <property name="text">
             <string> </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>130</width>
            <height>35</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_save">
            <property name="text">
             <string> </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_save">
            <property name="text">
             <string>Save as</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>14</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>act_frame:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_frameId">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1143</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ui_sources.clickable_label</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
